stacks/container-apps/root.yml

```diff
@@ -173,69 +175,13 @@ Resources:
         PlatformALBListenerPriorityPrefix: !Ref ExchangePlatformALBListenerPriorityPrefix
         EcrImageTag: !Ref ExchangeEcrImageTag
         SecretsVersion: !Ref ExchangeSecretsVersion
-        AppName: "exchange"
-        AltName: "api"
-        CreateWorker: "true"
-        DesiredWebsProduction: "5"
-        DesiredWorkersProduction: "6"
-        ErrorAlarmThreshold: "5"
-        ContainerPort: "3000"
-        HealthCheckPath: "/ping"
```

Comment:
Ahh, I see you made these defaults.  Which is still a bit weird.  But hey, this is mostly a copy/paste PR.

Reply:
The `AppName` param showed up in 29 places, which felt like too many to deal with right now, but FINE I'll do it